scene/resources/bone_map.cpp

```diff
@@ -152,7 +152,6 @@ void BoneMap::_validate_bone_map() {
 	} else {
 		bone_map.clear();
 	}
-	emit_signal("retarget_option_updated");
```

Comment:
@TokageItLab

Reply:
maybe I forgot replace it to `bone_map_updated`

Reply:
Verified. This deletion will not be a problem.